extensions/psx-info.el

```diff
@@ -171,7 +171,7 @@ not have to open the same file repeatedly.")
      (let* ((node (alist-get 'info-node args)))
        (format "info:%s" (symbol-name node))))))
 
-(p-search-def-property 'psx-info 'title #'psx-info--title)
+(p-search-def-property 'psx-info 'name #'psx-info--title)
```

Comment:
Should this be `psx-info--name`?

Reply:
This is a maybe confusing part of p-search. There are the two functions `p-search-def-property` and `p-search-def-field`. Any field defined by an extension package should indeed, as you mentioned, have a unique prefix. 

The properties on the other hand work a bit differently. Here, it's setting the name which the document will be displayed with in the results section. So `name` and `content` are the two _special_ properties regarding documents.